oe_theme.theme

```diff
@@ -1529,3 +1530,62 @@ function oe_theme_preprocess_page(array &$variables): void {
 function oe_theme_preprocess_pattern_organisation_teaser__preview(array &$variables): void {
   $variables['logo'] = ImageValueObject::fromArray($variables['logo']);
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function oe_theme_preprocess_oe_organisation(&$variables): void {
+  $entity = $variables['entity'];
+  // Prepare fields.
+  $fields_values = ['name' => 'name', 'oe_acronym' => 'acronym'];
+  foreach ($fields_values as $entity_field => $pattern_field) {
+    if (!$entity->get($entity_field)->isEmpty()) {
+      $variables[$pattern_field] = $entity->get($entity_field)->value;
+    }
+  }
+
+  // Extract the image if present.
+  if (!$entity->get('oe_logo')->isEmpty()) {
+    /** @var \Drupal\file\Plugin\Field\FieldType\FileItem $image_item */
+    $media_image = $entity->get('oe_logo')->entity;
+
+    if (!empty($media_image) && !$media_image->get('oe_media_image')->isEmpty()) {
```

Comment:
Why do you check here if image field is not empty? It is a required field in the image media bundle.

Reply:
It is safer. Media entity can be removed. In this case page will be unavailable due to thrown exception.